pkg/controller/plan/migration.go

```diff
@@ -1405,6 +1449,62 @@ func (r *Migration) updateConversionProgress(vm *plan.VMStatus, step *plan.Step)
 	return
 }
 
+func (r *Migration) updateConversionProgressEl9(pod *core.Pod, step *plan.Step) (err error) {
+	if pod.Status.PodIP == "" {
+		return
+	}
+
+	var disk_re = regexp.MustCompile(`v2v_disk_transfers\{disk_id="(\d+)"\} (\d{1,3}\.?\d*)`)
+	url := fmt.Sprintf("http://%s:2112/metrics", pod.Status.PodIP)
+	resp, err := http.Get(url)
+	if err != nil {
+		if strings.Contains(err.Error(), "connection refused") {
+			return nil
```

Comment:
why do we return with err = nil here? is it because we may get here before the metrics  endpoint is initialied?